podio.gemspec

```diff
@@ -19,16 +19,16 @@ Gem::Specification.new do |s|
 
   s.has_rdoc          = false
 
-  s.add_dependency('faraday', ['>= 0.8.0', '< 0.10.0'])
+  s.add_dependency('faraday', ['>= 0.8.0'])
   s.add_dependency('multi_json')
 
   if RUBY_VERSION < '1.9.3'
     s.add_dependency('i18n', '<= 0.6.11')
     s.add_dependency('activesupport', '~> 3.0')
     s.add_dependency('activemodel', '~> 3.0')
   else
-    s.add_dependency('activesupport', '>= 3.0')
-    s.add_dependency('activemodel', '>= 3.0')
+    s.add_dependency('activesupport', '>= 4.1.11')
```

Comment:
Btw why this need came for upgrading the active support version?
since we are using `>=` which means it will pick the latest version or the version defined in the `gemfile.lock`.